engine/src/main/resources/assets/i18n/menu_cs.lang

```diff
@@ -86,6 +86,7 @@
     "invert-y": "Invert Y Axis",
     "invert-z": "Invert Z Axis",
     "join-game-online": "Připojit se ke hře",
+    "join-server-refresh": "Refresh",
```

Comment:
"join-server-refresh" should be placed just after "join-server-action" - this is the case in some of your files but not others.